sys/parsoid.js

```diff
@@ -359,6 +359,15 @@ PSP.generateAndSave = function(restbase, req, format, currentContentRes) {
                             'summary' : 'definition', e);
                     }
                 }),
+                // MobileApps pre-generation
+                restbase.get({
+                    uri: new URI([rp.domain, 'v1', 'page', 'mobile-sections', rp.title]),
```

Comment:
One option to consider is to go directly to `/sys/movileapps/v1/handling/content/mobile-sections/no-cache/{title}` - like this we would avoid a request to title/revision, which we don't really need, as we've already verified that content is accessible while running the parsoid request. So, a bit less load on cassandra.